backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -97,6 +98,14 @@ public void getPatients_facilitySpecific_nullsAndSpecifiedFetchedAndSorted() {
         assertPatientList(patients, FRANK, BRAD, DEXTER, AMOS);
     }
 
+    @Test
+    @WithSimpleReportEntryOnlyUser
+    void addPatient_entryOnlyUser_error() {
+        assertSecurityError(
+                () -> _service.addPatient(null, null, "Fred", null, "Flintstone", "Jr.", LocalDate.of(1950, 1, 1), null,
```

Comment:
well this is an overwhelming and hard-to-parse method call, but not so urgently that it needs to be addressed immediately